packages/common/src/subchain/ReactSubchain.tsx

```diff
@@ -75,9 +81,15 @@ export function useQuery<T = any>(query: string): T {
         state.cachedClient = client;
         setCachedQuery(query);
         if (client?.subchain) {
-            state.cachedQueryResult = client.subchain.query(query);
+            state.cachedQueryResult = {
+                ...client.subchain.query(query),
+                isLoading: false,
+            };
         } else {
-            state.cachedQueryResult = null;
+            state.cachedQueryResult = {
+                isLoading: false,
```

Comment:
true; `client?.subchain` switches to non-null after wasm and state are loaded

Reply:
ah okay, so I will remove the error here